### PR TITLE
op-deployer: Add support for Isthmus calls to OPCM

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

**Review by blmalone — APPROVED** [review]
This lgtm, I'm not sure if we want to have an `apply_test.go` test for an Isthmus deploy - will leave that up to you.

**Comment by mslipper** [comment]
> This lgtm, I'm not sure if we want to have an apply_test.go test for an Isthmus deploy - will leave that up to you.

I'll need Maurelian's PR to do this. But yes - there will be a test!

**Pull request merged**
